Assets/Python/RFCUtils.py

```diff
@@ -1645,15 +1631,15 @@ def moveToClosestCity(self, unit):
 		
 	def evacuate(self, tPlot):
 		for tLoopPlot in self.surroundingPlots(tPlot):
-			for unit in utils.getUnitList(tLoopPlot):
-				lPossibleTiles = self.surroundingPlots(tLoopPlot, 2, lambda (x, y): utils.isFree(unit.getOwner(), (x, y), bNoEnemyUnit=True, bCanEnter=True))
-				tTargetPlot = utils.getRandomEntry(lPossibleTiles)
+			for unit in self.getUnitList(tLoopPlot):
+				lPossibleTiles = self.surroundingPlots(tLoopPlot, 2, lambda (x, y): self.isFree(unit.getOwner(), (x, y), bNoEnemyUnit=True, bCanEnter=True))
+				tTargetPlot = self.getRandomEntry(lPossibleTiles)
 				if tTargetPlot:
 					x, y = tLoopPlot
 					unit.setXY(x, y, False, True, False)
 			
 	def getWonderList():
-		return [i for i in range(iNumBuildings) if isWorldWonderClass(gc.getBuildingInfo(i).getBuildingClassType())]
+		return [i for i in range(iBeginWonders, iNumBuildings)]

```

Comment:
I prefer the implementation using `isWorldWonderClass` actually, it does not depend on the `iBeginWonders` constant being set correctly.